### PR TITLE
chore: Update jQuery v1.8 to v3.4.1

### PR DESCRIPTION
**Pull request by curbengh**

### CODE REVIEW

File: layout/_partial/google_analytics.ejs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-<script type="text/javascript">
+<script>
   var _gaq = _gaq || [];
   _gaq.push(['_setAccount', '<%= theme.google_analytics %>']);
   _gaq.push(['_trackPageview']);
 
   (function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+    var ga = document.createElement('script'); ga.async = true;
     ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
```
Comment (SukkaW) [inline layout/_partial/google_analytics.ejs]: `www.google-analytics.com` now supports HTTPS as well.
```suggestion
     ga.src = 'https://www.google-analytics.com/ga.js';
```

File: layout/_partial/head.ejs [hunk 2]
```diff
@@ -29,8 +29,7 @@
   <% if (page.keywords){ %><meta name="keywords" content="<%= page.keywords %>"><% } %>
 
   <link rel="alternate" href="<% if (theme.rss){ %><%- theme.rss %><% } else { %><%- config.root %>atom.xml<% } %>" title="<%= config.title %>" type="application/atom+xml">
-  <link rel="stylesheet" href="<%- config.root %>css/style.css" media="screen" type="text/css">
-  <!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8/jquery.min.js"></script>
+  <link rel="stylesheet" href="<%- config.root %>css/style.css" media="screen">
```
Comment (SukkaW) [inline layout/_partial/head.ejs]:
```suggestion
  <link rel="stylesheet" href="<%- url_for('css/style.css') %>" media="screen">
```

File: layout/_partial/after_footer.ejs [hunk 3]
```diff
@@ -16,9 +16,9 @@ var disqus_shortname = '<%= config.disqus_shortname %>';
 <% } %>
 
 <% if (theme.fancybox){ %>
-<link rel="stylesheet" href="<%- config.root %>fancybox/jquery.fancybox.css" media="screen" type="text/css">
+<link rel="stylesheet" href="<%- config.root %>fancybox/jquery.fancybox.css" media="screen">
```
Comment (SukkaW) [inline layout/_partial/after_footer.ejs]:
```suggestion
<link rel="stylesheet" href="<%- url_for('fancybox/jquery.fancybox.css') %>" media="screen">
```

Comment (curbengh) [reply layout/_partial/after_footer.ejs]: replaced with `css()` which integrates `url_for()`.

**Pull request merged**